src/Func.cpp

```diff
@@ -1150,9 +1151,9 @@ Stage Stage::specialize(Expr condition) {
     // The user may be retrieving a reference to an existing
     // specialization.
     const vector<Specialization> &specializations = definition.specializations();
-    for (size_t i = 0; i < specializations.size(); i++) {
-        if (equal(condition, specializations[i].condition)) {
-            return Stage(function, specializations[i].definition, stage_index, dim_vars);
+    for (const auto & specialization : specializations) {
```

Comment:
`const autio &specialization`1

Reply:
Just fixed the space. Didn't write `autio`, though. :wink: